_posts/Prakash-sa/Create 2022-11-17- Project Update: Draft Interface.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+author: Prakash-sa
+title: "Prakash's Final Project Interface"
+---
+
+# Final Project Interface Start
+
+For the final project, as stated in the inital project proposal, I am planning to do the data processing. I am going to do the web scraping of https://www.tradingview.com/. 
+I will do the Data Cleaning, Visualization and Storing with the python programming.
+
+I will clean and visualize the Stock market data from the different country. It will include the feature of the performance, rise and fall, Large cap, Small cap, Top gainers, Most active, etc.
+
+- For the data, I am using the HTML format from the website.
+
+I feel the following can be a good plan to create the project:
```

Comment:
Hi Prakash!  Instead of "can be" I would just say "is"

Reply:
It's so cool that you're scraping the data yourself!

Reply:
Thank's Hannah, I have done the changes.